### ISSUE TITLE
Mini-InternVL-4B-1.5 Finetuning Support & GPU requirements

### ISSUE THREAD

**Issue by babla9** [modelscope/swift#1050]
Thanks for supporting Mini-InternVL-4B-1.5!

Do you have finetuning support for this model? Do you know what the GPU requirement for it would be? Would 16gb GPU memory be sufficient?

**Comment by hjh0119** [comment]
Using gradient checkpointing and flash attention, it takes around 13GB for finetuning Mini-InternVL-4B-1.5 with coco-en-mini-2. If you use a dataset with larger images for training, the memory consumption will be slightly higher.

**Comment by babla9** [comment]
@hjh0119 thanks! Do you know if it supports text-only finetuning (eg no image provided)?

**Comment by hjh0119** [comment]
@babla9 The InternVL series models only support finetuning with images because the forward function requires `pixel_values` as input, which cannot be None. For more information, you can check [here](https://github.com/modelscope/swift/blob/main/docs/source_en/Multi-Modal/index.md)

**Comment by babla9** [comment]
Great, thanks so much for answering these questions!
